src/ServiceManagement/Compute/Commands.ServiceManagement.Test/FunctionalTests/LocationBasedReservedIPTests.cs

```diff
@@ -72,7 +74,9 @@ public void CreateReservedIPThenPaaSVM()
                 string reservedIpLabel1 = Utilities.GetUniqueShortName("ResrvdIPLbl", 5);
                 string reservedIpLabel2 = Utilities.GetUniqueShortName("ResrvdIPLbl", 5);
                 string dnsName = Utilities.GetUniqueShortName("Dns");
-                string deploymentName = Utilities.GetUniqueShortName("Depl");
+                string deploymentName1 = Utilities.GetUniqueShortName("Depl");
+                string deploymentName2 = Utilities.GetUniqueShortName("Depl");
```

Comment:
Maybe it's better to also have a different prefix, i.e. ```Dep2```, so that it's 100% unique, :-)